src/Microsoft.Extensions.Configuration.AzureAppConfiguration/TracingUtils.cs

```diff
@@ -79,8 +80,27 @@ public static string GetAssemblyVersion(string assemblyName)
         {
             if (!string.IsNullOrEmpty(assemblyName))
             {
-                // Return the version using only the first 3 fields and remove additional characters
-                return AppDomain.CurrentDomain.GetAssemblies().SingleOrDefault(assembly => assembly.GetName().Name == assemblyName)?.GetName().Version?.ToString(3).Trim('{', '}');
+                AssemblyInformationalVersionAttribute[] infoVersionAttributes = AppDomain.CurrentDomain.GetAssemblies().SingleOrDefault(assembly => assembly.GetName().Name == assemblyName)?
+                    .GetCustomAttributes(typeof(AssemblyInformationalVersionAttribute), false) as AssemblyInformationalVersionAttribute[];
```

Comment:
Why dont we use `GetCustomAttribute<AssemblyInformationalVersionAttribute>()` instead?

Reply:
Not sure what happened but I tried using that method originally and I just didn't see it show up, I must've been calling in the wrong place or something. Silly mistake, let me fix it